.swiftlint.yml

```diff
@@ -1,11 +1,7 @@
 excluded:
   - Carthage
-  - Pods
   - Examples
   - IntegrationTests
-  - PublicSDKAPITester
-  - vendor
-  - scan_derived_data
```

Comment:
I had this folder locally but it's no longer in git. Is it created by something I'm not aware of?

Reply:
we had it created in a lane in fastlane, don't know whether it's still around or not

Reply:
yep, `backend_integration_tests`

Reply:
Okay I'll leave this one then.